code/modules/mob/living/simple_animal/vore/rous.dm

```diff
@@ -41,6 +41,121 @@
 	vore_pounce_chance = 45
 	vore_icons = SA_ICON_LIVING | SA_ICON_REST
 
-/mob/living/simple_animal/hostile/rous/death()
+/mob/living/simple_animal/hostile/bigrat/death()
 	playsound(src, 'sound/effects/mouse_squeak_loud.ogg', 50, 1)
+	if(resting)
+		resting = 0
+		icon_state = icon_dead
+		update_icon()
 	..()
+
+/mob/living/simple_animal/hostile/bigrat/passive
+	name = "curious giant rat"
+	desc = "In what passes for a hierarchy among verminous rodents, this one is king. It seems to be more interested on scavenging."
+	follow_dist = 1
+	var/mob/living/carbon/human/food
+	var/hunger = 0
+
+/mob/living/simple_animal/hostile/bigrat/passive/Life()
+	. = ..()
+	if(!. || ai_inactive) return
+
+	for(var/obj/item/weapon/reagent_containers/food/snacks/S in oview(src,7)) //Accept thrown offerings and scavenge surroundings.
```

Comment:
Yikes, a little expensive. This will probably preclude these from ever being in mob spawners, given how expensive view is. Should also be `world.view` instead of 7 (or one of the SA-specific range vars).